### PR TITLE
Itr-1: Feat: initialize react app

### PR DESCRIPTION
**Pull request by MilaKolasava**
Added header and jumbotron.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import Header from './Header';
+import Jumbatron from './Jumbatron';
+
+
+function App() {
+  return (
+    <>
+        <Header />
+        <Jumbatron />
+
```
Comment (glebpigulevsky) [inline src/App.tsx]: extra empty line

File: src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Header from './Header';
+import Jumbatron from './Jumbatron';
+
+
+function App() {
+  return (
+    <>
+        <Header />
+        <Jumbatron />
```
Comment (glebpigulevsky) [inline src/App.tsx]: error in writing Jumbotron

File: src/Header.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
+.wrapper {
+    padding-top: 0.3rem;
+    margin: 0 auto;
+    width: 100%; 
+
```
Comment (glebpigulevsky) [inline src/Header.css]: extra empty line

File: src/Header.css [hunk 4]
```diff
@@ -0,0 +1,28 @@
+.wrapper {
+    padding-top: 0.3rem;
+    margin: 0 auto;
+    width: 100%; 
+    
+    background-color: black;
+    width: 100vw;
+
```
Comment (glebpigulevsky) [inline src/Header.css]: extra empty line

File: src/Header.css [hunk 5]
```diff
@@ -0,0 +1,28 @@
+.wrapper {
+    padding-top: 0.3rem;
+    margin: 0 auto;
+    width: 100%;
```
Comment (glebpigulevsky) [inline src/Header.css]: extra spaces. could you set up lint for your project?

File: src/Header.css [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+   
+   max-width: 100%;
+  }
+  .list-navigation {
+    display: flex;
+    column-gap: 20px;
+    list-style-type: none;
+    color: white;
+  }
+  .navigation {
```
Comment (glebpigulevsky) [inline src/Header.css]: empty line requires between style selectors

File: src/Header.css [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+    display: flex;
+    justify-content: space-between;
+      align-items: center;
+      font-size: 10px;
```
Comment (glebpigulevsky) [inline src/Header.css]: extra spaces

File: src/Header.tsx [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+      <nav className='navigation'>
+        <ul className='list-navigation'>
+          <li>HOME</li>
+          <li>BAND</li>
+          <li>TOUR</li>
+          <li>CONTACT</li>
+          <li>MORE</li>
+        </ul>
+                            <Loupe className='loupe' width='10' height='10' />
+            </nav>
```
Comment (glebpigulevsky) [inline src/Header.tsx]: extra spaces( it caoud be fixed with lint)

File: src/Jumbatron.tsx [hunk 9]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Jumbatron.css';
+
+function Jumbatron() {
+  return (
+    <div className='main-picture'>
+     
+    </div>
```
Comment (glebpigulevsky) [inline src/Jumbatron.tsx]:
```suggestion
    <div className='main-picture' />
```

File: src/Jumbatron.css [hunk 10]
```diff
@@ -0,0 +1,12 @@
+.main-picture {
+    
+    background: url(./Open-Air-Gampel.jpg), center, no-repeat;
+     
+    margin: 0 auto;
+   
+    width: 100vw;
+    height: 93vh;
```
Comment (glebpigulevsky) [inline src/Jumbatron.css]: calc(100vw - height of header)

File: src/Header.css [hunk 11]
```diff
@@ -0,0 +1,28 @@
+.wrapper {
+    padding-top: 0.3rem;
```
Comment (glebpigulevsky) [inline src/Header.css]: absolute unit

File: src/Header.css [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+    margin: 0 auto;
+    width: 100%; 
+    
+    background-color: black;
+    width: 100vw;
```
Comment (glebpigulevsky) [inline src/Header.css]: why do you have for one selector two width properties?

File: src/Header.css [hunk 13]
```diff
@@ -0,0 +1,28 @@
+.wrapper {
+    padding-top: 0.3rem;
+    margin: 0 auto;
```
Comment (glebpigulevsky) [inline src/Header.css]: why do you need margin 0 auto for element with width equal 100vw?

File: src/Header.tsx [hunk 14]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './Header.css';
+import {ReactComponent as Loupe} from './loupe.svg';
+
+function Header() {
+  return (
+    <div className='wrapper'>
```
Comment (glebpigulevsky) [inline src/Header.tsx]: I think this div is extra

**Review by glebpigulevsky — CHANGES_REQUESTED** [review]
lets make a clean code. project requires lint.

File: src/App.tsx (hunk 1 above)
Comment (MilaKolasava) [reply src/App.tsx]: done

File: src/Header.css (hunk 3 above)
Comment (MilaKolasava) [reply src/Header.css]: done

File: src/Header.css (hunk 4 above)
Comment (MilaKolasava) [reply src/Header.css]: done

File: src/Header.css (hunk 5 above)
Comment (MilaKolasava) [reply src/Header.css]: I deleted extra spaces, but my lint broke my app

File: src/Header.css (hunk 6 above)
Comment (MilaKolasava) [reply src/Header.css]: done

File: src/Header.css (hunk 7 above)
Comment (MilaKolasava) [reply src/Header.css]: done

File: src/Header.tsx (hunk 8 above)
Comment (MilaKolasava) [reply src/Header.tsx]: done

File: src/App.tsx (hunk 2 above)
Comment (MilaKolasava) [reply src/App.tsx]: done

File: src/Jumbatron.tsx (hunk 9 above)
Comment (MilaKolasava) [reply src/Jumbatron.tsx]: done

File: src/Header.tsx (hunk 14 above)
Comment (MilaKolasava) [reply src/Header.tsx]: Yes, you are right. I delete extra div.

File: src/Jumbatron.css (hunk 10 above)
Comment (MilaKolasava) [reply src/Jumbatron.css]: done, but I corrected height calc(100vh - ..)

File: src/Header.css (hunk 12 above)
Comment (MilaKolasava) [reply src/Header.css]: i fogot delet one of them, now I done

File: src/Header.css (hunk 13 above)
Comment (MilaKolasava) [reply src/Header.css]: deleted margin: 0 auto - it was redundant.

**Review by glebpigulevsky — APPROVED** [review]
good job!

**Pull request merged**